frontend/test/playwright/playwright.config.ts

```diff
@@ -54,7 +54,7 @@ const config: PlaywrightTestConfig = {
     baseURL,
     trace: "retain-on-failure",
   },
-  timeout: 60 * 1e3,
+  timeout: 10 * 1e3, // 5 seconds in enough to see if the test is stuck
```

Comment:
Did you mean...?
```suggestion
  timeout: 5 * 1e3, // 5 seconds in enough to see if the test is stuck
```

Reply:
Definitely! I tried 5 seconds, but the audio search page VR test needs more time.